spec/requests/v1/supplemental_claims_controller_request_spec.rb

```diff
@@ -77,6 +77,9 @@ def personal_information_logs
         # InProgressForm should be destroyed after successful submission
         in_progress_form = InProgressForm.find_by(user_uuid: user.uuid, form_id: '20-0995')
         expect(in_progress_form).to be_nil
+        # SavedClaim should be created with request data
+        saved_claim = SavedClaim::SupplementalClaim.find_by(guid: id)
```

Comment:
should we have tests to check if the evidence uploads are getting persisted in the way we expect?

Reply:
Added `expect` statements for this test and the other one